src/components/HistoryTables/MsigProposals/Proposal.tsx

```diff
@@ -107,19 +101,10 @@ export default function ProposalDetail(props: ProposalDetailProps) {
     stateError
   ])
 
-  const isProposer = useMemo(() => {
-    if (
-      !proposalFoundError &&
-      !!proposal &&
-      !!(props.walletAddress?.id || props?.walletAddress?.robust)
```

Comment:
I believe all these checks are already covered by `isAddrEqual` returning false in these cases